NebulaWorld/SimulatedWorld.cs

```diff
@@ -78,6 +91,16 @@ public static void UpdateRemotePlayerAnimation(PlayerAnimationUpdate packet)
             if (remotePlayersModels.TryGetValue(packet.PlayerId, out RemotePlayerModel player))
             {
                 player.Animator.UpdateState(packet);
+                player.Effects.UpdateState(packet);
+            }
+        }
+
+        public static void UpdateRemotePlayerLocalPlanetId(localPlanetSyncPckt packet)
+        {
+            if(remotePlayersModels.TryGetValue(packet.playerId, out RemotePlayerModel player))
+            {
+                player.localPlanetId = packet.planetId;
+                player.Effects.UpdatePlanet(packet.planetId);
             }
         }
```

Comment:
This function can be removed and the body moved to `UpdateRemotePlayerPosition` above since the `PlayerMovement` packet already contains the `LocalPlanetId`